Calculate/Calculator.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Calculate;
+    public class Calculator
+    {
+
+
+    public static double Add(double x, double y) => x + y;
+    public static double Subtract(double x, double y) => x - y;
+    public static double Multiply(double x, double y) => x * y;
+    public static double Divide(double x, double y) => x / y;
+
+    public IReadOnlyDictionary<char, Func<double, double, double>> MathematicalOperations { get; } =
+    new Dictionary<char, Func<double, double, double>>
+    {
+            { '+', (a, b) => a + b },

```

Comment:
```suggestion
            { '+', Add },

```
much simpler way of doing it

Reply:
this also applies to the rest

Reply:
Ah thanks, that makes sense!